### ISSUE TITLE
Some visitors think Liberapay is only about software

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#348]
Reported by @MartinDelille in #345.

**Comment by MartinDelille** [comment]
Yes I think we should make it look like as broad as possible even if it is primaly designed for creator. The notion of common is also not so well-known from the public, it might required a specific page: "what are the commons?"
